sdk/communication/Azure.Communication.Sms/tests/samples/Sample1_SmsClient.cs

```diff
@@ -13,23 +13,37 @@ namespace Azure.Communication.Sms.Tests.samples
     /// <summary>
     /// Samples to send sms
     /// </summary>
-    public partial class Sample1_SmsClient
+    public partial class Sample1_SmsClient : SamplesBase<SmsClientTestEnvironment>
     {
         [Test]
         [SyncOnly]
         public void SendSms()
         {
             SmsClient smsClient = CreateSmsClient();
 
-            #region Snippet:SendSms
-            /// Send an sms
-            SmsSendResult sendSmsResult = smsClient.Send(
-                from: "<leased-phone-number>",
-                to: "<to-phone-number>",
-                message: "<message-to-send>");
-
-            Console.WriteLine("MessageId: " + sendSmsResult.MessageId);
-            #endregion Snippet:SendSms
+            #region Snippet:Azure_Communication_SmsClient_Send_Troubleshooting
+            try
+            {
+                #region Snippet:Azure_Communication_SmsClient_Send
+                SmsSendResult result = smsClient.Send(
+                    //@@ from: "+18001230000", // Phone number acquired on your Azure Communication resource
+                    //@@ to: "+18005670000", // recipient's phone-number
+                    /*@@*/ from: TestEnvironment.PhoneNumber,
+                    /*@@*/ to: TestEnvironment.PhoneNumber,
+                    message: "Hello"); // message-to-send
```

Comment:
you can also add an emoji like 👋🏻 to the message to show that we support emojis as well 🙂